src/scss/components/_media-element.scss

```diff
@@ -123,7 +123,7 @@ $mediaElement-spacing           : spacing(x2);
 
         .c-mediaElement--fullstack & {
             flex: auto;
-            margin-top: spacing(x0.5);
+            margin-top: (spacing(x2) * -1);
```

Comment:
I'd leave this and try applying a class like `u-spacingTop--large` to the element you're looking to style.

Reply:
You mean create a new one that is similar @DamianMullins -because I need it to be `-16px`?

Reply:
In that can you use the `-#{spacing(x2)}` syntax to keep it consistent with other negative values?

Also, remember that this class isn't just used in Menu Web, it could be in use anywhere that Fozzie is used in a project. So it's probably worth creating a modifier for this style so it doesn't break layouts anywhere else.

Reply:
Also, do you want the negative spacing across all device widths?